docs/resources/sql_endpoint.md

```diff
@@ -39,6 +39,8 @@ The following arguments are supported:
 * `enable_serverless_compute` - Whether this SQL endpoint is a Serverless endpoint. To use a Serverless SQL endpoint, you must enable Serverless SQL endpoints for the workspace.
 * `channel` block, consisting of following fields:
   * `name` - Name of the Databricks SQL release channel. Possible values are: `CHANNEL_NAME_PREVIEW` and `CHANNEL_NAME_CURRENT`. Default is `CHANNEL_NAME_CURRENT`.
+* `disable_uc` - Whether this endpoint will have Unity Catalog disabled.
```

Comment:
This field was marked as "public undocumented" in API. Do we want to expose this field?

Reply:
I can remove it from the documentation, but I remember that somebody has asked about it

Reply:
@yongwang-db @yuchenhuo Can you confirm if this field is something we want to expose as public API?